test/ProfileTracerTest.cpp

```diff
@@ -44,6 +46,9 @@ void ProfileTracerTest::SetUp(void)
         m_data.push_back({time, rank, event, region_hash});
         time += 1.0;
     }
+
+     m_data.push_back({40, 0, geopm::EVENT_SHORT_REGION, 0xdeadbeef});
+     m_data.push_back({41, 1, geopm::EVENT_EPOCH_COUNT, region_hash});
```

Comment:
```suggestion
     m_data.push_back({41, 1, geopm::EVENT_EPOCH_COUNT, 1});
```
The epoch count event should reflect a number times the epoch event has occurred.